docs/logs/getting-started-console/Program.cs

```diff
@@ -17,33 +17,38 @@
 using Microsoft.Extensions.Logging;
 using OpenTelemetry.Logs;
 
-namespace SourceGeneration;
-
-public class Program
+using var loggerFactory = LoggerFactory.Create(builder =>
 {
-    public static void Main()
+    builder.AddOpenTelemetry(options =>
```

Comment:
Rename `options` to `logging` similar to #4821? We could use this style for the other examples as well.